use-cases.md

```diff
@@ -192,22 +193,24 @@ We have already used `log_event_with_payload` in the client span use case. Event
     span.log_event('cache-miss') 
 ```
 
-The tracer automatically records a timestamp of the event, in contrast with tags that apply to the entire span. In the future versions of the API it will be possible to associate an externally provided timestamp with the event, e.g. see [pull request #38](https://github.com/opentracing/opentracing-go/pull/38).
+The tracer automatically records a timestamp of the event, in contrast with tags that apply to the entire Span. In the future versions of the API it will be possible to associate an externally provided timestamp with the event, e.g. see [pull request #38](https://github.com/opentracing/opentracing-go/pull/38).
```

Comment:
we can remove PR reference, as it's been merged, and just do an example of generic log method